subjects/localhost/audit/README.md

```diff
@@ -0,0 +1,63 @@
+#### Localhost is about creating your own HTTP server and test it with an actual browser.
+##### Take the necessary time to understand the project and to test it, looking into the source code will help a lot.
+
+### Basic server mechanics
+
+#### The student should be able to justify his choices and explain the following:
+###### How does an HTTP server works?
```

Comment:
I understand, this is why I added a line before, I didn't want to have oversized questions that include 90% of the same content all the time...
Maybe a good solution could be to change this sentence:
`**The student should be** able to justify his choices and explain the following:`
By this one:
`**Is the student able** to justify his choices and explain the following:`

This will give a clear yes/no output to the following questions while keeping them small in size and clear to understand.

What do you think, does it seems clear to you? Otherwise I'll go for the long version on each question.

Reply:
yes, it looks better, but there is a problem as well...
The student can be able to explain one or more of those points without actually doing it on the project so we need to find a way to make him show what he did to the auditer.
maybe we can add something like:
```**Note:** ask the student to show you the work he did as he explain it```

Reply:
done